### ISSUE TITLE
Hero guides not loading when using this mod

### ISSUE THREAD

**Issue by Grofff** [rossengeorgiev/dota2mod_manacost#7]
First of all I can't thank you enough for this great mod!

I'm having an issue where hero guides won't load after the first match played when I'm using your mod. If I remove 
                      ''Game				costmod_combined
			Game				costmod_damagetype
			Game				costmod_goldcost
			Game				costmod_manacost''

from gameinfo.gi everything goes back to normal and the guides load. I've already tried verifying game cache and everything is ok. The only solution to get the guides working was to remove the lines above. I can post my gameinfo.gi if you need it.

i3 4150
4GB RAM
AMD 7750 1GB
Win 10 x64

**Comment by rossengeorgiev** [comment]
Yep. This is a known issue and there is no workaround. Maybe Valve will fix it one day

**Comment by Grofff** [comment]
Damn, I hoped there was something wrong on my end. I really doubt Valve will do anything about it.
